src/Api/Jobs/JobsHostedService.cs

```diff
@@ -41,6 +41,11 @@ public override async Task StartAsync(CancellationToken cancellationToken)
             .StartNow()
             .WithCronSchedule("0 30 */12 * * ?")
             .Build();
+        var everyDayAtTenPmTrigger = TriggerBuilder.Create()
```

Comment:
`everyDayAtTenPmTrigger` this variable name is timezone specific, and I'm concerned it might lead to some confusion. 

What do you think about naming the trigger based on the intended job? 

Something like `smTrashCleanupTrigger`.

Reply:
Sounds good, renamed in [3c22cf4](https://github.com/bitwarden/server/pull/2863/commits/3c22cf4c8fb3b7ca98eb27ccc097726571630bea)